apps/site/assets/css/_header.scss

```diff
@@ -476,7 +478,7 @@ nav.m-menu {
       @include heading-4();
       color: $white;
       &:hover, &:active {
-        background-color: $brand-primary-darkest;
+        background-color: $brand-primary;
```

Comment:
Since we're setting it to the same color it was when not hovered, let's use `inherit` for the value.